request/graphql/schema/types/scalars.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2023 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package types
+
+import (
+	"encoding/hex"
+
+	"github.com/sourcenetwork/graphql-go"
+	"github.com/sourcenetwork/graphql-go/language/ast"
+)
+
+var BytesScalarType = graphql.NewScalar(graphql.ScalarConfig{
+	Name:        "Bytes",
+	Description: "The `Bytes` scalar type represents an array of bytes.",
+	Serialize: func(value any) any {
+		switch value := value.(type) {
+		case []byte:
+			return hex.EncodeToString(value)
```

Comment:
question: Why are you encoding/decoding at all here (including in ParseValue)? 

todo: The `graphql` library lacks any documentation here, but we can add some to our code to document why `Serialize` and `ParseValue` exist.